src/main/java/seedu/duke/Deadline.java

```diff
@@ -0,0 +1,57 @@
+package seedu.duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Deadline task.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate dueDate;
+    protected String dueTime;
+
+    /**\
+     * Creates a Deadline task that has a date but no time included.
+     * @param description description of Deadline.

```

Comment:
Great and consistent work on the Javadoc documentation throughout your submission! 

Although, it might be better to leave a blank line between the description and the `@param` so that it is more aligned to the convention. Similar comments for the rest of the documentation as well.

Please note that you might have accidentally included an additional backslash `\` in line 14 as well.